frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -423,6 +530,49 @@ export default class NewForm extends Vue {
   currentTeamsGroupAlias = this.teamsGroupAlias;
   titlePlaceHolder = '';
 
+  openCollapsedIfRequired() {
+    const formData = new FormData(this.formRef);
+
+    let basicInfoCollapsed = this.basicInfoRef.classList.contains('collapsed');
+    let logisticsCollapsed = this.logisticsRef.classList.contains('collapsed');
+    let scoringRulesCollapsed = this.scoringRulesRef.classList.contains(
+      'collapsed',
+    );
+
+    console.log('FORM DATA');
+    for (const [key, value] of formData.entries()) {
+      console.log(key, value);
+      const isEmpty = value === '';
+      if (isEmpty) {
```

Comment:
```suggestion
      if (value !== '') {
        continue;
      }
```

acá se puede usar el [guard clause pattern](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html).